spec/controllers/pets_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.describe PetsController, type: :controller do
+
+  before do
+    @request.env['devise.mapping'] = Devise.mappings[:user]
+  end
+
+  describe '里親募集登録' do
+    let(:user) do
+      create(:user, email: 'test123456789@test.com', password: 'password12345', password_confirmation: 'password12345', &:confirm)
+    end
+    let!(:params) do
+      {
+        pet_form: {
+          photos: ['dog2.jpeg']
+        },
+        pet: {
+          category: 'dog',
+          petname: 'riku',
+          age: 12,
+          gender: 'male',
+          classification: 'Chihuahua',
+          introduction: 'おとなしく、賢い',
+          castration: 'neutered',
+          vaccination: 'vaccinated',
+          recruitment_status: 0,
+          user_id: user.id
+        }
```

Comment:
enum の指定は文字列ではなくシンボルにするよう癖付けておくと良いかもです。

Reply:
シンボルにするよう心掛けます。

Reply:
シンボルに直すとこんな感じで書けます。
速度面を考えるとこっちの方が効率いいんだな、程度で覚えておくといいかもです。

```suggestion
        pet: {
          category: :dog,
          petname: :riku,
          age: 12,
          gender: :male,
          classification: :Chihuahua,
          introduction: 'おとなしく、賢い',
          castration: :neutered,
          vaccination: :vaccinated,
          recruitment_status: 0,
          user_id: user.id
        }
```